packages/patternlab/styleguide/source/assets/js/index.js

```diff
@@ -26,7 +26,9 @@ import keywordSearch from "./modules/keywordSearch.js";
 import linkListToggle from "./modules/linkListToggle.js";
 import listAfterFloat from "./modules/listAfterFloat.js";
 import locationListing from "./modules/locationListing.js";
```

Comment:
Are these being fully replaced? If so, can we clean up these no longer used files?